.travis.yml

```diff
@@ -6,8 +6,10 @@ install: gem install jekyll
 
 script: jekyll build
 
+# Ignoring YAMLLintBear because it is treating indentation as whitespace
+# Indentation is required
 env:
-  global:
-  - NOKOGIRI_USE_SYSTEM_LIBRARIES=true 
+  global:  # Ignore YAMLLintBear
```

Comment:
the idea with those comments is that you can do something like `# Ignore YAMLLintBear because ...` in one go, not to have like 3 comments everywhere.

You should file an issue at coala-bears about this.